src/app.ts

```diff
@@ -71,6 +76,64 @@ const main = async () => {
     res.json({ success: 'true' });
   });
 
+  const storage = multer.memoryStorage();
+  const upload = multer({ storage: storage });
+
+  // Not really sure where in the code base go, but here's the working route!
+  // I did my part :)
+  // Is just using `/flyers/` the right route here?
+  app.post('/flyers/', upload.single('file'), async (req, res) => {
+    // Ensure there is an image file present
+    if (!req.file) {
+      return res.status(400).json({ error: 'No file uploaded' });
+    }
+    const {
+      categorySlug,
+      endDate,
+      flyerURL,
+      location,
+      organizationID,
+      startDate,
+      title,
+    } = req.body;
+
+    // Is there a better way to do this? (assert request body has required fields)
+    if (
+      categorySlug == null ||
+      endDate == null ||
+      flyerURL == null ||
+      location == null ||
+      organizationID == null ||
+      startDate == null ||
+      title == null
+    ) {
+      return res.status(400).json({ error: 'Missing a required field' });
+    }
+
+    // Get the file from form-data and await the upload service
+    const imageB64 = req.file.buffer.toString('base64');
+    const imageURL = await utils.uploadImage(imageB64);
+
+    // Find the organization related to the request
+    const organization = await OrganizationModel.findById(new ObjectID(organizationID));
```

Comment:
You should use the Organization Controller here instead, per MVC design pattern

Reply:
my bad, didn't realize this existed!